admin/admin-service/src/main/java/io/mosip/admin/packetstatusupdater/dto/ApplicantVerficationDto.java

```diff
@@ -0,0 +1,13 @@
+package io.mosip.admin.packetstatusupdater.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ApplicantVerficationDto {
+
+    private String applicantPhoto;
+
+    private String dateOfBirth;
```

Comment:
Can we make this dto flexible to allow countries to configure what all applicant details they need here ?

Reply:
taken care

Reply:
Resolved